libpointmatcher_ros/src/point_cloud.cpp

```diff
@@ -52,6 +52,7 @@ namespace PointMatcher_ros
 			else if (name == "rgb" || name == "rgba")
 			{
 				descLabels.push_back(Label("color", (name == "rgba") ? 4 : 3));
+				fieldTypes.push_back(PM_types::DESCRIPTOR);
```

Comment:
To be more consistent this should be below the next line.

Reply:
Should i do that?

Reply:
I've missed your message, i've done it right now.